src/app/state/staking/saga.test.ts

```diff
@@ -158,21 +158,65 @@ describe('Staking Sagas', () => {
         dump_timestamp_iso: '2022-03-23T00:52:41.337Z',
         list: [
           {
-            rank: 1,
-            entityAddress: 'oasis1qq3xrq0urs8qcffhvmhfhz4p0mu7ewc8rscnlwxe',
-            name: 'stakefish',
-            nodeAddress: 'oasis1qrg52ccz4ts6cct2qu4retxn7kkdlusjh5pe74ar',
-            escrow: '0.2',
-            status: true,
+            active: true,
+            current_commission_bound: {
+              epoch_end: 0,
+              epoch_start: 0,
+              lower: 0,
+              upper: 20000,
+            },
+            current_rate: 5000,
+            entity_address: 'oasis1qq3xrq0urs8qcffhvmhfhz4p0mu7ewc8rscnlwxe',
+            entity_id: 'eZuacXy5s3/nolB/E3gF4vqUYdvfOlVaaBXGfZcGwKc=',
+            escrow: {
+              active_balance: '153866511910432557',
+              active_shares: '109032182592723438',
+              debonding_balance: '16650100374694405',
+              debonding_shares: '16650100374694405',
+              num_delegators: 7763,
+              self_delegation_balance: '619388791323275',
+              self_delegation_shares: '438908447022946',
+            },
+            in_validator_set: true,
+            media: {
+              name: 'stakefish',
+            },
+            node_id: 'SQZZd1wsWXdFsqswUoh6hZtmzu+ejuSnrGeHtgIBJDo=',
+            rank: 2,
+            start_date: '2024-04-11T03:43:47Z',
```

Comment:
Changes in API binding can be removed when start_date is removed from mocks, but I think it's worth keeping more valid bindings anyway. With old generated types are mark as Date which is not true, these props are strings.